modular_skyrat/modules/modular_vending/code/vending.dm

```diff
@@ -14,8 +14,38 @@
 	if(skyrat_contraband)
 		contraband += skyrat_contraband
 
+	/// Time to make clothes amounts consistent!
+	for (var/item in products)
+		if(products[item] < 5 && ispath(item, /obj/item/clothing) && skyrat_IsNotGameplay(item))
+			products[item] = 5
+	for (var/item in premium)
+		if(premium[item] < 5 && ispath(item, /obj/item/clothing) && skyrat_IsNotGameplay(item))
+			premium[item] = 5
+			
 	QDEL_NULL(skyrat_products)
 	QDEL_NULL(skyrat_premium)
 	QDEL_NULL(skyrat_contraband)
 	return ..()
 
+/obj/machinery/vending/proc/skyrat_IsNotGameplay(var/obj/item/clothing/clothing_path)
```

Comment:
That proc name doesn't follow the convention of `snake_case`, which would mean it should be looking like `skyrat_is_not_gameplay()`

Furthermore, that name is very ambiguous, and it shouldn't contain `skyrat` at the start of it, it's entirely unnecessary.

You also don't need to (and shouldn't) add `var` to the typepaths of proc parameters, it's already implied and handled the same way by the compiler.

Lastly, I would appreciate if you documented your proc, aka, added a comment starting with `/// ` on the line before that explain what this proc is supposed to do.

Reply:
- Oop, I looked at another proc and made assumptions, sorry!
- Yea, I think a better name would be `is_item_increase_banned` or something to that degree- and the `skyrat_` part was me once again assuming from `skyrat_products` and such!
- I already knew this, yet I still did it anyways- oops-
- Will do!
Thanks for taking time to review my coding travesty, pff-